training/Dockerfile.prod

```diff
@@ -0,0 +1,35 @@
+# pull base image
+FROM python:3.9-slim-bookworm
```

Comment:
Is it possible to condense the two dockerfiles into one and have docker-compose ymls specify the configurations that separate dev/prod?

Reply:
Yea that seems reasonable